classes/migration/upgrade/v3_6_0/I857_ContributorRolesTypes.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_6_0/I857_ContributorRolesTypes.php
+ *
+ * Copyright (c) 2025 Simon Fraser University
+ * Copyright (c) 2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class I857_ContributorRolesTypes
+ *
+ * @brief OMP specific actions related to contributor roles and types
+ */
+
+namespace APP\migration\upgrade\v3_6_0;
+
+use Illuminate\Support\Facades\DB;
+use PKP\install\DowngradeNotSupportedException;
+use PKP\migration\Migration;
+
+class I857_ContributorRolesTypes extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // Remove isVolumeEditor from author settings
+        DB::table('author_settings')
+            ->select(['setting_name'])
+            ->where('setting_name', '=', 'isVolumeEditor')
+            ->delete();
+    }
```

Comment:
I think we need to consider this setting in migration, to set the contributor role to EDITOR if this setting is true.
It seems the user group of an contributor could be author, but also this setting could exist.
So, if user group of an existing contributor is not volume editor (i.e. there is no EDITOR contributor role already set in parent::up()) and this setting is true, then add EDITOR as contributor role as well.